index.html

```diff
@@ -5,9 +5,40 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <title>Document</title>
+    <title>Portfolio</title>
 </head>
 <body>
-    <h1>hello</h1>
+    <div id="header-background-mobile">
+      <header id="Toolbar-Dark-Compact">
+         <div id="Logo-Placeholder">
+             <p id="My-Logo">Qwerty</p>
+         </div>
```

Comment:
- [ ] Kindy ensure your logo is wrapped in an `a` tag, this is for the user experience as the ogo acts as a button to take the users to the homepage.